Assets/Scripts/UI/ExperienceBar.cs

```diff
@@ -31,6 +30,7 @@ private void Start()
 
     private void UpdateSlider(int pTargetValue)
     {
+        Debug.Log("Updaing Slider");
```

Comment:
I feel like I'm a mierenneuker for pointing this out but Updating isn't spelled correctly 🤓